recipes/perl-tk/804.034/meta.yaml

```diff
@@ -0,0 +1,163 @@
+    - perl-encode
+    - perl-extutils-makemaker
+    - perl-test-simple
+    - libpng
+    - xorg-libx11
+    - xorg-libxft
+    - libjpeg-turbo
+    # - libfreetype  - disabled, set XFT=0 in Makefile
+
+  run:
+    - perl
+    - perl-encode
+    - perl-test-simple
+    - tk # [not osx]
+    - xorg-libx11 # [not osx]
+
+test:
+  # Perl 'use' tests
+  imports:
+    - Tk
+   #  - Tk::Adjuster
+   #  #- Tk::Adjuster::Item
+   #  - Tk::After
+   #  - Tk::Animation
+   #  - Tk::Balloon
+   #  - Tk::Bitmap
+   #  - Tk::BrowseEntry
+   #  - Tk::Button
+   #  - Tk::Canvas
+   #  - Tk::Checkbutton
+   #  - Tk::Clipboard
+   #  - Tk::CmdLine
+   #  - Tk::ColorDialog
+   #  - Tk::ColorEditor
+   #  - Tk::ColorSelect
+   #  - Tk::Compound
+   #  - Tk::Configure
+   #  - Tk::Derived
+   #  - Tk::Dialog
+   #  - Tk::DialogBox
+   #  - Tk::DirTree
+   #  #- Tk::DirTreeDialog
+   #  - Tk::Dirlist
+   #  - Tk::DragDrop
+   #  - Tk::DragDrop::Common
+   #  #- Tk::DragDrop::Local
+   #  - Tk::DragDrop::Rect
+   #  - Tk::DragDrop::SunConst
+   #  - Tk::DragDrop::SunDrop
+   #  - Tk::DragDrop::SunSite
+   #  #- Tk::DragDrop::Win32Drop
+   #  #- Tk::DragDrop::Win32Site
+   #  - Tk::DragDrop::XDNDDrop
+   #  - Tk::DragDrop::XDNDSite
+   #  - Tk::DropSite
+   #  - Tk::DummyEncode
+   #  #- Tk::DummyEncode::X11ControlChars
+   # # - Tk::DummyEncode::iso8859_1
+   #  - Tk::English
+   #  - Tk::Entry
+   #  - Tk::ErrorDialog
+   #  - Tk::Event
+   #  - Tk::Event::IO
+   #  - Tk::FBox
+   #  - Tk::FileSelect
+   #  - Tk::FloatEntry
+   #  - Tk::Font
+   #  - Tk::Frame
+   #  - Tk::HList
+   #  - Tk::IO
+   #  - Tk::IconList
+   #  - Tk::Image
+   #  - Tk::InputO
+   #  - Tk::ItemStyle
+   #  - Tk::JPEG
+   #  - Tk::LabEntry
+   #  - Tk::LabFrame
+   #  #- Tk::LabRadiobutton
+   #  - Tk::Label
+   #  #- Tk::LabeledEntryLabeledRadiobutton
+   #  - Tk::Labelframe
+   #  - Tk::Listbox
+   #  - Tk::MMtry
+   #  - Tk::MMutil
+   #  - Tk::MainWindow
+   #  - Tk::MakeDepend
+   #  - Tk::Menu
+   #  #- Tk::Menu::Button
+   #  #- Tk::Menu::Cascade
+   #  #- Tk::Menu::Checkbutton
+   #  - Tk::Menu::Item
+   #  #- Tk::Menu::Radiobutton
+   #  #- Tk::Menu::Separator
+   #  - Tk::Menubar
+   #  - Tk::Menubutton
+   #  - Tk::Message
+   #  - Tk::MsgBox
+   #  - Tk::Mwm
+   #  - Tk::NBFrame
+   #  - Tk::NoteBook
+   #  - Tk::Optionmenu
+   #  - Tk::PNG
+   #  - Tk::Pane
+   #  - Tk::Panedwindow
+   #  - Tk::Photo
+   #  - Tk::Pixmap
+   #  - Tk::Pretty
+   #  - Tk::ProgressBar
+   #  - Tk::ROText
+   #  - Tk::Radiobutton
+   #  - Tk::Region
+   #  - Tk::Reindex
+   #  - Tk::ReindexedROText
+   #  - Tk::ReindexedText
+   #  - Tk::Scale
+   #  - Tk::Scrollbar
+   #  - Tk::Spinbox
+   #  - Tk::Stats
+   #  - Tk::Submethods
+   #  - Tk::TList
+   #  - Tk::Table
+   #  - Tk::Text
+   #  - Tk::Text::Tag
+   #  - Tk::TextEdit
+   #  - Tk::TextList
+   #  - Tk::TextUndo
+   #  - Tk::Tiler
+   #  - Tk::TixGrid
+   #  - Tk::Toplevel
+   # - Tk::Trace
+    # - Tk::Tree
+    # - Tk::Widget
```

Comment:
Can you now, include these tests again? Or should they be removed?

Reply:
okay, will try